server/src/main/webapp/WEB-INF/rails/package.json

```diff
@@ -111,8 +111,8 @@
     "speed-measure-webpack-plugin": "^1.5.0",
     "stats-webpack-plugin": "^0.7.0",
     "style-loader": "^2.0.0",
-    "stylelint": "^16.8.1",
```

Comment:
Or you could update `stylelint-webpack-plugin` instead.

ref https://github.com/webpack-contrib/stylelint-webpack-plugin/releases/tag/v5.0.0

Reply:
Thanks - but unfortunately cannot, as need to do a webpack 5 upgrade first (#10852)